05-FITS/io_fits_ascii_overview.ipynb

```diff
@@ -165,21 +168,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "f[1].header['observer'] = \"Edwin Hubble\"\n",
-    "    \n",
+    "f[extnum].header['observer'] = \"Edwin Hubble\"\n",
+    "\n",
     "# Assigning a tuple to a keyword splits\n",
     "# the tuple into a keyword value and comment.\n",
+    "# FITS header keyword is also not case-sensitive.\n",
     "    \n",
-    "f[1].header['NOBS'] = (2, \"Number of observing nights\")\n",
-    "print(\"observer:\", f[1].header['observer'])\n",
-    "print(f[1].header.comments['NOBS'], f[1].header['NOBS'])"
+    "f[extnum].header['NOBS'] = (2, \"Number of observing nights\")\n",
+    "print(\"observer:\", f[extnum].header['OBSERVER'])\n",
+    "print(f[extnum].header.comments['NOBS'], f[extnum].header['NOBS'])"
```

Comment:
I feel we ought to mention `header.insert` when one cares about the ordering of keywords.